include/bluetooth/scan.rst

```diff
@@ -80,40 +80,28 @@ Filter modes
 |             | Otherwise, the not found callback is called.                                    |
 +-------------+---------------------------------------------------------------------------------+
 
-Directed advertising
+Directed Advertising
 ====================
 
-To support receiving directed advertising, it is required to enable either `CONFIG_BT_PRIVACY <zephyr:CONFIG_BT_PRIVACY>` or `BT_SCAN_WITH_IDENTITY <zephyr:BT_SCAN_WITH_IDENTITY>`.
-It is recommnended to use a privace feature.
-Using scan with identity should be used only when scan with privacy is not possible.
+To support receiving Directed Advertising packets with the Scanning Module, enable one of the following options in Zephyr:
```

Comment:
Another requirement:
When privacy is enabled directed advertising is only supported from bonded peers.